common/db/migrations/079_change_job_type_to_non_enum.rb

```diff
@@ -0,0 +1,25 @@
+Sequel.migration do
+
+  up do
+    alter_table(:job) do
+      add_column(:job_type, String, :null => false)
+    end
+
+    self[:job].update(:job_type => self[:enumeration_value].filter(:id => :job_type_id).select(:value))
+
+    alter_table(:job) do
+      drop_foreign_key(:job_type_id)
+    end
+
```

Comment:
Move DDL commands up to top then wrap the data updates in self.transaction do ... end